content/03.methods.md

```diff
@@ -293,8 +294,28 @@ Table S1. Clinical metadata collected for OpenPBTA. {#tbl:S1}
 
 #### Molecular Subtyping
 
-The `molecular_subtype` column in the `pbta-histologies.tsv` file contains molecular subtype information derived as described below.
-Medulloblastoma subtypes SHH, MYC, Group 3, and Group 4 were predicted using an [RNA expression classifier](https://github.com/d3b-center/medullo-classifier-package) on the RSEM FPKM data.
+The `molecular_subtype` column in the `pbta-histologies.tsv` file contains molecular subtype information derived as described below, following World Health Organization 2016 classification criteria [@doi:10.1007/s00401-016-1545-1].
+
+Medulloblastoma (MB) subtypes SHH, MYC, Group 3, and Group 4 were predicted using an [RNA expression classifier](https://github.com/d3b-center/medullo-classifier-package) on the RSEM FPKM data.
+
+High-grade glioma (HGG) subtypes were derived using the criteria below (additional details in the analysis [README](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-HGG)):
+1. If any sample contained an H3F3A K28M, HIST1H3B K28M, HIST1H3C K28M, or HIST2H3C K28M mutation and no BRAF V600E mutation, it was subtyped as `DMG, H3K28`.
```

Comment:
Hmm, yeah, since they are describing protein, they would be in non-italicized caps. Here, I wrote out each gene to be very specific, but we could opt for saying `H3 K28M` etc, like in [this paper](https://braintumor.org/wp-content/assets/WHO-Central-Nervous-System-Tumor-Classification.pdf). Even in this paper, though, they seem to bounce back and forth with italicizing and not. I would suggest we just adhere to the [human gene/protein naming standards](https://www.gmb.org.br/geneprotein-nomenclature-guidelines) of cap/italic for gene names/RNA, and caps for protein. (I thought there was another official website, but couldn't find quickly - could find fly, mouse, zebrafish, ha!).